config/dev.exs

```diff
@@ -96,21 +96,28 @@ unless IEx.started?() do
     ]
 end
 
+amqp_connection =
+  if System.get_env("USE_LOCAL_RABBIT_TLS") do
```

Comment:
No need for a `if`. We put the default value as 2nd parameter, so if the env value is not set it will be used

Reply:
Do you mean provide the whole connection string as default argument? My idea was to have a simple "switch" without passing explicitly the whole string but just a boolean thing to switch between the two

Reply:
Yes, my fault. I read it to fast. I didn't realize that this is just a boolean.
All good!